scripts/util/calc_tpm.py

```diff
@@ -0,0 +1,45 @@
+def main(counts_name_path, gene_lengths_path, outfile_path, has_header):
+    counts_file = open(counts_name_path, "r")
+    counts = {}
+    if has_header:
+        counts_file.readline()
```

Comment:
Can this be auto-detected?

Reply:
if we are comfortable assuming:
> if row 1 column 2 is not an `int` it has a header

The corner case is if someone provides a sample name like `1234`